module-1/dist/assets/js/admin-page.js

```diff
@@ -53,13 +53,21 @@ replyButtons.forEach(elem => {
                 subject: `re: ${message}`,
                 text: text.value 
             }
-            postData(`${hostUrl}sendEmail`, data)
+            postData(`${HOST_URL}sendEmail`, data)
             .then(data => { 
                 // TODO process data
             })
             .catch((error) => {
                 throw new Error(error)
             })
+            form.reset()
+            form[0].value=""
+            form[0].placeholder = "Your mail has been sent"
+            form[0].classList.add('color__placeholder')
+            function formClose() {
```

Comment:
Don't change styles in JS, instead add or remove certain css class and have an appropriate rules in css

Reply:
Maybe, I don't understand correctly but we did as you describe. Adding a style at the string 66, 'color__placeholder' is a certain class in the style-contacts.css